interactive_select_printer_test.go

```diff
@@ -12,12 +12,12 @@ import (
 
 func TestInteractiveSelectPrinter_Show(t *testing.T) {
 	go func() {
-		keyboard.SimulateKeyPress(keys.Down)
```

Comment:
Hi, instead of changing that test, could you please make a new one? Thanks!

Reply:
Sure, I was on the fence on that one. Reverted to your original test + a new one for the "maxheight sliding window" logic.